internal/pkg/otel/samples/darwin/platformlogs_hostmetrics.yml

```diff
@@ -0,0 +1,37 @@
+receivers:
```

Comment:
Another version of the same file as `logs_host_metrics.yml`? :thinking:

Reply:
removed the first one